Rubberduck.VBEEditor/SafeComWrappers/VBA/VBComponent.cs

```diff
@@ -11,14 +14,49 @@ namespace Rubberduck.VBEditor.SafeComWrappers.VBA
 {
     public class VBComponent : SafeComWrapper<VB.VBComponent>, IVBComponent
     {
+        private const int MaxRetryCount = 10;
+
+        private enum ComErrorCodes : uint
```

Comment:
Technically HRESULT are signed integers, with negative values indicating an error. I would use `int`, not `uint` here.

Reply:
I know that they are integers. However, I could not find a way to assign the literal if the enum type is `int`. I think we do the same for window messages.

Reply:
`unchecked((int)0x80004005)`